test/helper/user.js

```diff
@@ -0,0 +1,94 @@
+import Promise from 'bluebird';
+import UserModel from 'models/User';
+import User from 'common/User';
+import { createAndFindBooks } from 'test/helper/book';
+
+const defaultAccount = {
+  mail: 'dragonball@jump.com',
+  password: 'wakuwakusitekitazo',
+};
+
+function createUser(account = defaultAccount) {
+  const user = new UserModel(account);
+  return new Promise((resolve)=> {
+    user.save(resolve);
+  });
+}
+
+function createUsers(numberOfUsers = 10) {
+  const users = Array.from(new Array(numberOfUsers), (value, index) => index);
+  return Promise.reduce(users, (total, userIndex)=> {
+    return new Promise((resolve)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).